CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide/experimental/BoxFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements.uncollide.experimental;
+
+import au.gov.asd.tac.constellation.graph.GraphReadMethods;
+
+/**
+ * 
+ * @author Nova
+ */
+class BoxFactory {
```

Comment:
Add access modifiers

Reply:
Deliberately left the class at Default/Package private access level.
If the class were public it would need to reside somewhere different as a general graph utility class..Currently there is no other use case and given it is a very simple concept I believe a class fulfilling this role would already have been created if such a use case was likely to exist. Therefore it should remain package private.

Reply:
I have made the create method protected.